assets/css/base/_base.scss

```diff
@@ -22,3 +24,45 @@ h3 {
 p {
   font-size: 1em;
 }
+
+// Links and Buttons
+a {
+  color: $blue-dark;
+  font-weight: bold;
```

Comment:
As was the case ion the typography pull request, this should be a numeric value. It might be a good idea to add variables for font weights to the variables file, so for example you could use `$bold` instead of having to remember that bold is 700. Does that make sense @TeddTech?

Reply:
Yeah, it does. I think I'll go ahead with that method